src/colorPrinter.py

```diff
@@ -57,3 +57,12 @@ def addstr(self, y, x, text, attr=None):
             attr = self.currentAttributes
 
         self.screen.addstr(y, x, text, attr)
+
+    # perhaps there's a more elegant way to do this
+    def clearseg(self, y, startX, endX):
```

Comment:
was torn between camel-case and trying to follow the addstr() styling, switched to clearSegment as I agree clearseg is not clear